core/src/main/java/hudson/cli/CLICommand.java

```diff
@@ -304,6 +284,20 @@ public int main(List<String> args, Locale locale, InputStream stdin, PrintStream
         }
     }
 
+    private void logAndPrintErrorWithDefaultValues(List<String> args, Throwable e) {
```

Comment:
It's unclear what "default values" refers to.

Reply:
Thanks for the feedback! I'll change the name to something more specific.